### DISCUSSION TITLE
How can you update the S3 access tokens

### DISCUSSION THREAD

**Discussion by amos-baron** [vmware-tanzu/velero#3462] [Community support Q&A] [answered]
After installing Velero on the cluster, is it possible to change the S3 access tokens, what secret are they saved in?

**Comment by ashish-amarnath** [comment, marked as answer]
Yes, it is possible to update the credentials after Velero is installed. The credentials to the backup storage location are stored in the `cloud-credentials` secret in the `velero` namespace.
